src/contracts/business-center/business-center.spec.ts

```diff
@@ -202,208 +199,208 @@ describe('Business Center', function test() {
       });
 
       it('rejects an invite of a member', async () => {
-        const invitePromise = executor.executeContractTransaction(
-          currentBc, 'invite', { from: accounts[0], autoGas: 1.1 }, accounts[1],
+        const invitePromise = runtimes[0].executor.executeContractTransaction(
+          currentBc, 'invite', { from: identity0, autoGas: 1.1 }, identity1,
         );
         expect(invitePromise).to.be.rejected;
       });
 
       it('allows to change the join Schema', async () => {
-        await executor.executeContractTransaction(
-          currentBc, 'setJoinSchema', { from: accounts[0], autoGas: 1.1 }, 1,
+        await runtimes[0].executor.executeContractTransaction(
+          currentBc, 'setJoinSchema', { from: identity0, autoGas: 1.1 }, 1,
         );
       });
 
       it('does not allow to change the join schema', async () => {
-        const setJoinSchemaPromise = executor.executeContractTransaction(
-          currentBc, 'setJoinSchema', { from: accounts[2], autoGas: 1.1 }, 1,
+        const setJoinSchemaPromise = runtimes[1].executor.executeContractTransaction(
+          currentBc, 'setJoinSchema', { from: identity1, autoGas: 1.1 }, 1,
         );
         expect(setJoinSchemaPromise).to.be.rejected;
       });
 
       it('allows to leave the business center', async () => {
-        await executor.executeContractTransaction(
-          currentBc, 'cancel', { from: accounts[2], autoGas: 1.1 },
+        await runtimes[1].executor.executeContractTransaction(
+          currentBc, 'cancel', { from: identity1, autoGas: 1.1 },
         );
-        const isMember = await executor.executeContractCall(
-          currentBc, 'isMember', accounts[2], { from: accounts[2], gas: 3000000 },
+        const isMember = await runtimes[0].executor.executeContractCall(
+          currentBc, 'isMember', identity1,
         );
         expect(isMember).to.be.false;
       });
     });
 
     describe('when working with a InviteOnly Business Center', async () => {
       it('rejects a join', async () => {
-        const joinPromise = executor.executeContractTransaction(
-          currentBc, 'join', { from: accounts[2], autoGas: 1.1 },
+        const joinPromise = runtimes[1].executor.executeContractTransaction(
+          currentBc, 'join', { from: identity1, autoGas: 1.1 },
         );
         expect(joinPromise).to.be.rejected;
       });
 
       it('adds a member, when a member is invited', async () => {
-        await executor.executeContractTransaction(
-          currentBc, 'invite', { from: accounts[0], autoGas: 1.1 }, accounts[1],
+        await runtimes[0].executor.executeContractTransaction(
+          currentBc, 'invite', { from: identity0, autoGas: 1.1 }, identity1,
         );
-        const isMember = await executor.executeContractCall(currentBc, 'isMember', accounts[1]);
+        const isMember = await runtimes[0].executor.executeContractCall(currentBc, 'isMember', identity0);
```

Comment:
```suggestion
        const isMember = await runtimes[0].executor.executeContractCall(currentBc, 'isMember', identity1);
```
should be identity1 because he is invited